src/main/resources/alfresco/transforms/0100-basePipelines.json

```diff
@@ -210,6 +252,17 @@
         "pdfRendererOptions",
         "imageMagickOptions"
       ]
+    },
+    {
+      "transformerName": "OutlookMsg2Office",
```

Comment:
Please follow the name pattern invented by Erik. In this case it would be "outlookMsgToLibreofficeViaText"

Reply:
cc @eknizat

Reply:
This is has been changed now